dev/com.ibm.ws.jpa.tests.spec10.injection.mdb_fat/fat/src/com/ibm/ws/jpa/tests/spec10/injection/mdb/RepeatWithJPA32.java

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
Please add a copyright. All java source files must have a copyright. I see this was copied from another file that did not have a copyright, so ideally it would be best to add the copyright to that file as well (though the year should be the year it was delivered - see git history)